common/src/main/java/com/sequenceiq/cloudbreak/ccm/cloudinit/CcmV2ParameterConstants.java

```diff
@@ -14,6 +14,10 @@
 
     public static final String CCMV2_AGENT_KEY_ID = "ccmV2AgentKeyId";
 
+    public static final String CCMV2_AGENT_MACHINE_USER_ACCESS_KEY = "ccmV2AgentMachineUserAccessKey";
```

Comment:
Nit: Perhaps appending `Id` to the value (and `_ID` to the constant name) would express the purpose more precisely.

Reply:
ok